benchmarks/dynamo/pr_time_benchmarks/benchmark_base.py

```diff
@@ -58,7 +104,10 @@ def print(self):
     def collect_all(self):
         self.results = []
         if self._instruction_count:
-            self.results.append(
-                (self.name(), "instruction_count", self.count_instructions())
+            r = self.count_instructions()
+            self.results.append((self.name(), "instruction_count", r))
+            scribe_log_torch_benchmark_compile_time(
```

Comment:
I am unsure if we shall log here. When we run A/B this will just send both to scribe.